docs/website/docs/tutorial/rest-api.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Load data from a REST API
+description: How to extract data from a REST API using dlt's generic REST API source
+keywords: [tutorial, api, github, duckdb, rest api, source, pagination, authentication]
+---
+
+This tutorial shows how to extract data from a REST API using dlt's generic REST API source. The tutorial will guide you through the basics of setting up and configuring the source to load data from the API into a destination.
+
+As a practical example, we'll build a data pipelines that loads data from the [Pokemon](https://pokeapi.co/) and [GitHub](https://docs.github.com/en/rest) APIs into a [DuckDB](https://duckdb.org) database.
+
+## What you will learn
+
+- How to set up a REST API source
+- Configuration basics for API endpoints
+- Handling pagination, authentication
+- Configuring the destination database
+- Relationships between different resources
+- How to append, replace, and merge data in the destination
+- Loading data incrementally by fetching only new or updated data
+
+## Prerequisites
+
+- Python 3.9 or higher installed
+- Virtual environment set up
+
+## Installing dlt
```

Comment:
I think I would just put this into the prerequisites category vs having it's own section.